protobuf-format.md

```diff
@@ -0,0 +1,88 @@
+# Protobuf Event Format for CloudEvents  - Version 0.1
+
+## Abstract
+
+The Protocol Buffers Format for CloudEvents (CE) defines
+the encoding of events in the protocol buffers binary format.
+
+## Status of this document
+
+This document is a working draft.
+
+## 1. Introduction
+
+CloudEvents is a standardized and transport-neutral definition of the
+structure and metadata description of events. This specification
+defines how the [Context
+Attributes](spec.md#context-attributes) defined in the CloudEvents in
```

Comment:
This sentence is a bit awkward. Perhaps:
This specification defines how the [Context Attributes](spec.md#context-attributes) defined in the CloudEvents specification MUST be encoded in the protocol buffer binary format.

Reply:
Thanks for the improved wording, I will use this phrasing.